src/controls/generic-controls.js

```diff
@@ -289,25 +289,31 @@ export default class GenericControls {
     }
 
     const KEY_BINDINGS = {
-      KeyW: 'moveForward',
+      w: 'moveForward',
+      W: 'moveForward',
       ArrowUp: 'moveForward',
```

Comment:
Don't the arrow keys need to be replaced with numbers or similar as well? 

Good if we can keep them working, they are really important to the controller use case...